src/components/summaryBox.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+function SummaryBox() {
+  return (
+    <>
+    <link href="https://fonts.googleapis.com/css?family=Poppins&display=swap" rel="stylesheet" />
```

Comment:
we don't use font at specific point like this.

Reply:
i'll make the necessary changes